.github/pull_request_template.md

```diff
@@ -11,6 +11,7 @@ Feel free to delete sections of the template which do not apply to your PR, or a
 - [ ] Any updated packages successfully build (or no packages were changed)
 - [ ] All package sources are available
 - [ ] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`)
+- [ ] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
```

Comment:
I think it should still be merged to `dev` here since its in `1.0-dev`.  It seems like it'd cause more headaches if we had commits on `main` that are not reflected in our `latest` branches.